### DISCUSSION TITLE
Blueman cannot continue

### DISCUSSION THREAD

**Discussion by AwsSmadii** [blueman-project/blueman#2279] [Q&A]
blueman:
BlueZ:
Distribution: Debian
Guys the bluetooth keeps shutting down every time I stop the video, I am using Kali linux

**Comment by cschramm** [comment]
So what's the exception?

**Comment by AwsSmadii** [comment]
The Bluetooth discontinues when I stop a YouTube video for 15 seconds, without any reason

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cschramm** [comment]
I assume with blueman not being able to continue, you indicate that the blueman-applet process stops. There must be some exception or at a least return code from it.

**Comment by AwsSmadii** [comment]
Oh yes, this is what is tells me in this image .

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AwsSmadii** [comment]
Hi, has anything happened?
here is the text if it is hard to open the picture
Connection to Bluez failed, Bluez daemon is not running, bluetooth-manager
cannot continue, This probably means that there were no Bluetooth adapters
detected or Bluetooth daemon was not started.

‫في الجمعة، 23 فبراير 2024 في 8:01 م تمت كتابة ما يلي بواسطة ‪Aws Smadi‬‏ <‪
***@***.***‬‏>:‬

**Comment by cschramm** [comment]
Sounds like your BlueZ daemon vanished. Check its logs ( `sudo journalctl -u bluetooth.service`).
